### DISCUSSION TITLE
Coins are not coming from the exchange

### DISCUSSION THREAD

**Discussion by Hardmod31** [Chia-Network/chia-blockchain#18493] [Support]
Good day!
Txn Hash
0xb19f39e456c5e687aee8da5d9b366b0b670460e519505e1cffa3853b36525b25
8 days have passed, the coins are still not displayed on the wallet balance.

**Comment by BrandtH22** [comment]
Hey @Hardmod31 , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
- If using the cli then I would stop chia, rename the wallet db folder in ~.chia\mainnet\wallet\db then restart chia.

If the funds are still not visible can you let us know:

- if the wallet icon in the top right is orange or green?
- if orange can you click on the wallet icon and let us know if you see any connected peers?
- can you provide your log file from ~.chia\mainnet\log\debug.log?

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
